cosmo/monitors/dark_data_models.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import os
+from glob import glob
+from monitorframe.datamodel import BaseDataModel
+
+from ..filesystem import find_files, data_from_exposures
+from .. import SETTINGS
+
+FILES_SOURCE = SETTINGS['filesystem']['source']
+
+
+class DarkDataModel(BaseDataModel):
+    cosmo_layout = False
+    program_id = ['15771', '15533/', '14940/', '14520/', '14436/', '13968/', '13521/', '13121/', '12716/', '12423/',
+                  '11895/']
+    # subdir_pattern = '?????'
+
+    def get_new_data(self):
+        header_request = {
+            0: ['ROOTNAME', 'SEGMENT'],
+            1: ['EXPTIME', 'EXPSTART']
+        }
+
+        table_request = {
+            1: ['PHA', 'XCORR', 'YCORR', 'TIME'],
+            3: ['TIME', 'LATITUDE', 'LONGITUDE']
+        }
+
+        files = []
+
+        for prog_id in self.program_id:
+
+            new_files_source = os.path.join(FILES_SOURCE, prog_id)
+            files += find_files('*corrtag*', data_dir=new_files_source)
```

Comment:
```suggestion
            files += find_files('*corrtag*', data_dir=new_files_source, subdir_pattern=None)
```
This is why you were getting empty `DataFrame`s. The default `subdir_pattern` is `'?????'` which would only match if the directories you were looking under also had subdirectories with a five-character name, and that's where the files were. Since you're looking under specific subdirectories of the cosmo cache, the `subdir_pattern` argument just needs to be set to `None` to find the files in that those "flat" directories